style/_overrides.scss

```diff
@@ -10,3 +10,7 @@
 .sd-sphinx-override p {
   margin-top: 0;
 }
+
+.sd-sphinx-override:focus-visible {
+  outline-offset: 0.25rem;
```

Comment:
This helps ensure that this extension's focus rings are accessible. If the focus ring hugs the element, if there is no gap, and the background or border color of the element matches the focus ring, then it will be hard to perceive the focus ring, and therefore not accessible. 

For reference, see WCAG 2.2 Understanding Docs - [Understanding SC 2.4.13:
Focus Appearance (Level AAA)](https://w3.org/TR/WCAG22#focus-appearance)